client-tools/Import/ImportHelper.cpp

```diff
@@ -971,7 +976,7 @@ void ImportHelper::addLastField(char const* field, size_t fieldLength,
   // add --merge-attributes arguments
   if (!_mergeAttributesInstructions.empty()) {
     for (auto& [key, value] : _mergeAttributesInstructions) {
-      if (row == _rowsToSkip) {
+      if (row == _rowsToSkip && !_headersSeen) {
         std::for_each(
             value.begin(), value.end(),
             [this, key = &key](Step const& attrProperties) {
```

Comment:
Shouldn't we rather capture `key` by reference here, instead of taking its address?
```suggestion
            [this, &key](Step const& attrProperties) {
```